driver.sh

```diff
@@ -155,11 +193,12 @@ mako_reactor() {
   KBUILD_BUILD_TIMESTAMP="Thu Jan  1 00:00:00 UTC 1970" \
   KBUILD_BUILD_USER=driver \
   KBUILD_BUILD_HOST=clangbuiltlinux \
-  make -j"${jobs:-$(nproc)}" CC="${CC}" HOSTCC="${CC}" LD="${LD}" HOSTLD="${HOSTLD:-ld}" AR="llvm-ar-9" "${@}"
+  make -j"${jobs:-$(nproc)}" CC="${CC}" HOSTCC="${CC}" LD="${LD}" HOSTLD="${HOSTLD:-ld}" AR="${AR}" "${@}"
 }
 
 build_linux() {
-  CC="$(command -v ccache) $(command -v clang-9)"
+  # Wrap CC in ccache
+  CC="$(command -v ccache) ${CC}"
```

Comment:
can we move this ccache check above?

Reply:
`ccache` isn't a required binary, that's why it is here instead of up there. I'll make that a comment.